libraries/chain/chain_controller.cpp

```diff
@@ -1224,15 +1227,25 @@ void chain_controller::initialize_chain(chain_initializer_interface& starter)
          for (int i = 0; i < 0x10000; i++)
             _db.create<block_summary_object>([&](block_summary_object&) {});
 
+         signed_block block{};
```

Comment:
Please add a comment to indicate what you are doing here.

Reply:
Added comment:
`         // create a dummy block and cycle for our dummy transactions to send to applied_irreversible_block below
`